backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/utils/JsonConverters.java

```diff
@@ -69,4 +72,22 @@ public ExternalId fromJsonOrNull(@Nullable final JsonExternalId jsonExternalId)
 		}
 		return ExternalId.of(jsonExternalId.getValue());
 	}
+
+	@NonNull
+	public JsonAttachmentType toJsonAttachmentType(@NonNull final AttachmentEntryType type)
```

Comment:
add a unit test that checks that we have an answer for each value of AttributeEntryType.values()